packages/mui-joy/src/styles/CssVarsProvider.tsx

```diff
@@ -40,6 +40,7 @@ const { CssVarsProvider, useColorScheme, getInitColorSchemeScript } = createCssV
   },
   defaultColorScheme: 'light',
   prefix: 'joy',
+  shouldSkipVar: (keys) => keys[0] === 'typography',
```

Comment:
Can we be more descriptive here? Maybe `shouldSkipGeneratingVars`?

Reply:
Yep, that is much clearer.